dictionary.txt

```diff
@@ -930,3 +930,4 @@ _rp	₎
 \daleth	ℸ	
 ^epsilon	ᵋ	
 \rightangle	∟	
+\starequals	≛	
```

Comment:
Suggestion: remove the `s`. I was able to find references to a LaTeX shortcut called `\starequal` but not `\starequals`

http://www.sput.nl/software/latex-symbols.html

```suggestion
\starequal	≛	
```